lib/Services/RoutingService.php

```diff
@@ -50,6 +50,7 @@ public static function call(string $controllerClassname, bool $authenticated = t
         $serverRequest = ServerRequestFactory::fromGlobals();
         $response = $controller($serverRequest);
 
+        # TODO sspv2 return Symfony\Component\HttpFoundation\Response (Template instance) in SSP v2
```

Comment:
Is this something to fix before releasing these namespace changes?

Reply:
No, this is just to mark a return type needed when we do an overhaul of the module to use Symfony DI, in SimpleSAMLphp v2. Now we are still using our own container and resolving instances ourselves.